raft.go

```diff
@@ -620,7 +620,7 @@ func (r *Raft) run() {
 // runFollower runs the FSM for a follower.
 func (r *Raft) runFollower() {
 	didWarn := false
-	r.logger.Printf("[INFO] raft: %v entering Follower state", r)
+	r.logger.Printf(`[INFO] raft: %v entering Follower state (Leader: "%s")`, r, r.Leader())
```

Comment:
! I haven't committed all of the fmt verbs to memory yet.  Super useful, thanks.